Screens/LanguageScreen.js

```diff
@@ -0,0 +1,89 @@
+
+import React, {Component} from 'react';
+import { StyleSheet, Text,Image, View,TouchableOpacity} from 'react-native';
+import {storeSettings}from '../Actions/settingsActions';
+import DeviceInfo from "react-native-device-info";
+import { connect } from 'react-redux';
+import {setLanguage} from 'redux-i18n';
+
+let deviceId =
```

Comment:
Deviceid is located in store
https://github.com/net375/theater-schedule-mobile/pull/9/files#diff-676fef3a117000527d17e01850f60842R31
get it from there